scripts/cleanup

```diff
@@ -34,6 +34,7 @@ killall netmaster
 etcdctl rm -recursive /contiv > /dev/null 2>&1
 etcdctl rm -recursive /contiv.io > /dev/null 2>&1
 curl -X DELETE http://localhost:8500/v1/kv/contiv?recurse > /dev/null 2>&1
+curl -X DELETE http://localhost:8500/v1/kv/contiv.io?recurse > /dev/null 2>&1
```

Comment:
do you also want to cleanup the skydns related keys here, like what you are doing in `testbedAPI.py`? It would be nice if we could use common cleanup script.